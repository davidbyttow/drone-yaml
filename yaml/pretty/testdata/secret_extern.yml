kind: secret

external_data:
  username:
    path: secrets/data/docker
    name: username

  password:
    path: secrets/data/docker
    name: password
