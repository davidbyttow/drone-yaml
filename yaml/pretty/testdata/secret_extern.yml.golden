---
kind: secret
type: external
external_data:
  password:
    path: secrets/data/docker
    name: password
  username:
    path: secrets/data/docker
    name: username

...
